CONTRIBUTING.md

```diff
@@ -199,5 +199,22 @@ $ yarn start --disable-host-check --public=$(echo ${server.dev-server} | sed -e
 ```
 
 ### Che server a.k.a WS master
-
-To be provided soon.
+There is a [devfile](https://github.com/eclipse/che/blob/master/devfile.yaml) for development of Che server in Che.
+To build Che one may run a predefined build task from the devfile.
+
+Starting Che master requires some manual steps.
+Open a terminal in runtime container (`che-server-runtime`) and perform:
+ - First, set `CHE_HOME` environment variable with absolute path to parent folder of Che master's Tomcat.
+   It might look like `/projects/che/assembly/assembly-main/target/eclipse-che-*-SNAPSHOT/eclipse-che-*-SNAPSHOT`.
```

Comment:
While reading it I expect that `CHE_HOME=/projects/che/assembly/assembly-main/target/eclipse-che-*-SNAPSHOT/eclipse-che-*-SNAPSHOT` but looks like it's not.
The following bash works fine for me. Can we include it to our che-in-che devfile?
![Screenshot_20190816_130210](https://user-images.githubusercontent.com/5887312/63160288-17acc680-c026-11e9-81d1-7552bcf20adb.png)